web-app/src/main/java/com/infoshare/webapp/service/FileJSONService.java

```diff
@@ -0,0 +1,37 @@
+package com.infoshare.webapp.service;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.infoshare.webapp.model.Questions;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Objects;
+
+@Service
+public class FileJSONService {
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    private List<Questions> questions;
+
+    public void loadFile(MultipartFile file) throws IOException {
+        if (Objects.equals(file.getContentType(), "application/json")) {
+            questions = objectMapper.readValue(new InputStreamReader(file.getInputStream()), new TypeReference<List<Questions>>() {});
+        }
+    }
+
+    public List<Questions> getQuestions() {
+        return questions;
+    }
+    public void setQuestions(List<Questions> questions){
+        this.questions = questions;
+    }
```

Comment:
Moim zdaniem jeśli do listy questions nie będzie potrzebny dostęp z zewnątrz to setter i getter nie jest konieczny

Reply:
ok, usunąłem. Pozostałość po tym jak szukałem rozwiązania dla tej funkcjonalności